apps/hubot/dev.env

```diff
@@ -0,0 +1,6 @@
+ROCKETCHAT_URL_VALUE=https://chat.pathfinder.gov.bc.ca
```

Comment:
this could be a different url in dev and test environments than the production one.